src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Invariant.cs

```diff
@@ -27,7 +27,7 @@ internal static unsafe int InvariantIndexOf(string source, string value, int sta
             }
         }
 
-        internal static unsafe int InvariantIndexOf(ReadOnlySpan<char> source, ReadOnlySpan<char> value, bool ignoreCase)
+        internal static unsafe int InvariantIndexOf(ReadOnlySpan<char> source, ReadOnlySpan<char> value, bool ignoreCase, bool start = true)
```

Comment:
Why does this now take `start` if it doesn't use it?

Reply:
Right thanks.